openedx_ledger/api.py

```diff
@@ -114,6 +118,7 @@ def create_transaction(
                     "content_key": content_key,
                     "parent_content_key": parent_content_key,
                     "content_title": content_title,
+                    "course_run_start_date": course_run_start_date,
```

Comment:
Can we add a unit test to assert that course run date is successfully being saved against a transaction just to be safe?